.eslintrc.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:ember-suave/recommended',
+  ],
+  env: {
+    qunit: true,
+    node: true,
+  },
+  globals: {
+    // 'expectAssertion': true,
+    // 'expectDeprecation': true,
+    // 'expectNoDeprecation': true,
+    // 'expectWarning': true,
+    // 'expectNoWarning': true,
+    // 'ignoreAssertion': true,
+    // 'ignoreDeprecation': true,
+
+    // Electron
+    'requireNode': true,
+    'processNode': true,
+
+    // A safe subset of "browser:true":
+    // 'window': true,
+    // 'document': true,
+    // 'setTimeout': true,
+    // 'clearTimeout': true,
+    // 'setInterval': true,
+    // 'clearInterval': true,
+
+    // 'Symbol': true,
+    // 'WeakMap': true,
+  },
+  rules: {
+    'comma-dangle': ['error', 'always-multiline'],
+    'ember-suave/no-const-outside-module-scope': 'off',
+    'ember-suave/no-direct-property-access': 'off',
+    'ember-suave/require-access-in-comments': 'off',
+    'newline-before-return': 'error',
+    'new-cap': 'off',
```

Comment:
Pretty much the reason why I started this. Our own guidelines are far more strict than this (based on AirBnB), but do not fit the Ember community, so I think this is a solid compromise while stile providing some consistency. 

There is stuff like [prefer-arrow-callback](http://eslint.org/docs/rules/prefer-arrow-callback) which I would gladly see here in all electron files, but won't work for Ember code, so I'm not sure if we should include it here. Still, having a very modern node project gives us a lot of nice syntax.